Makefile: use LDFLAGS as provided by the user

In some circumstances, the user may want to pass some LDFLAGS, like
-L flags to point to the zlib location if it was not installed in a
standard location.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN a/mkfs.axfs-legacy/Makefile b/mkfs.axfs-legacy/Makefile
--- a/mkfs.axfs-legacy/Makefile
+++ b/mkfs.axfs-legacy/Makefile
@@ -6,7 +6,7 @@
 all:   mkfs.axfs
 
 mkfs.axfs: $(MKFSOBJS)
-	$(CC) $(CFLAGS) -o mkfs.axfs $(MKFSOBJS) -lz
+	$(CC) $(CFLAGS) $(LDFLAGS) -o mkfs.axfs $(MKFSOBJS) -lz
 
 clean_mkfs.axfs:
 	rm -rf $(MKFSOBJS) mkfs.axfs
